docs/src/pages/components/lists/lists.md

```diff
@@ -93,3 +93,6 @@ It renders 200 rows and can easily handle more.
 Virtualization helps with performance issues.
 
 {{"demo": "pages/components/lists/VirtualizedList.js"}}
+
+We encourage the usage of [react-window](https://github.com/bvaughn/react-window) when possible.
```

Comment:
https://github.com/bvaughn/react-window#how-is-react-window-different-from-react-virtualized